README.md

```diff
@@ -1,14 +1,13 @@
-This readme is a skeleton, please fill me out with some useful info!
-
 # fulfilment-lambdas
-This is a set of lambdas in a step function to allow us to fulfil from zuora.
+This project generates newspaper fulfilment files for the Home Delivery and Guardian Weekly products, currently sold [here](https://subscribe.theguardian.com).
 
-They are built in ES6, using [StandardJS](https://standardjs.com) and [Flow](http://flow.org).
+The file generation process is written as an [AWS Step Function](https://aws.amazon.com/step-functions/), which queries the subscription data in Zuora and constructs files based on the results.
 
-It is automatically built using teamcity and riffraff as CODE or PROD as appropriate.
+These files are uploaded to Salesforce, so that various distributors can download them. The upload process is carried out via a separate lambda (salesforce_uploader), and is triggered by a scheduled CloudWatch event (i.e. it is not part of the Step Function).
```

Comment:
Since you're here... what do people think of this approach? I think it would be more robust (and easier to monitor) if the upload task was contained within the Step Function definition

Reply:
I think @pvighi did this so that we had time to check the files before they're uploaded, but we could probably achieve the same thing by adding another wait task to the Step Function... And then when we have some fully automated QA/validation steps we could replace the wait with those instead

Reply:
Yup and we could always nest the step function so that the uploader steps can be triggered independently without having to do any generation first.